packages/suite/src/hooks/wallet/useRbfForm.ts

```diff
@@ -34,16 +35,18 @@ const getBitcoinFeeInfo = (info: FeeInfo, feeRate: string) => {
 
 const getEthereumFeeInfo = (info: FeeInfo, gasPrice: string) => {
     const current = new BigNumber(gasPrice);
-    const minFee = current.plus(1);
+    const minFee = new BigNumber(fromWei(info.levels[0].feePerUnit, 'gwei'));
```

Comment:
I see 3 use cases.

1. User created tx with low fee because he wanted to save some money but now he wants to speed tx up. 
gas price of tx 1GWEI, current is 12 GWEI, ok ✅ 
![Screenshot 2023-06-27 at 12 04 38](https://github.com/trezor/trezor-suite/assets/33235762/83f2e8f4-21bb-4c74-a4f6-79bbb4d10f36)
2. User used normal fee rate, our recommended. However, gas price increased in the next block. ok (same as 🔝 ) ✅ 
3. User used normal fee rate, our recommended. However, for some reason we recommended wrong gas price. E.g. tx created with 12 GWEI gas price, I want to bump it, we still think that 12 GWEI is gas price. We offer to use 12 GWEI again. ❌ 
![Screenshot 2023-06-27 at 12 00 36](https://github.com/trezor/trezor-suite/assets/33235762/5b3af581-284f-4214-be52-025e9fcb3e77)
![Screenshot 2023-06-27 at 14 44 31](https://github.com/trezor/trezor-suite/assets/33235762/528b9a4f-e7b5-4a58-afc5-b7ba683dbec0)

So I would add something like - if current gas price is same as gas price of tx, increase it by one.

Reply:
Done

Reply:
`console.log(new BigNumber(fromWei(info.levels[0].feePerUnit, 'gwei')).toString());`
![Screenshot 2023-06-28 at 10 30 08](https://github.com/trezor/trezor-suite/assets/33235762/05af0116-36cf-48c7-9791-dcfd56d977a5)
![Screenshot 2023-06-28 at 10 30 11](https://github.com/trezor/trezor-suite/assets/33235762/ee7974af-723e-4813-b82a-e2515037e60e)

Reply:
When did that happen? Works on my device for Ethereum mainnet and also Goerli testnet.

<img width="752" alt="Screenshot 2023-06-28 at 12 10 35 PM" src="https://github.com/trezor/trezor-suite/assets/66002635/5a066b89-c8ff-4c02-800f-a3d559ba5e49">

<img width="745" alt="Screenshot 2023-06-28 at 12 12 11 PM" src="https://github.com/trezor/trezor-suite/assets/66002635/55d05086-ee21-4673-80c0-ebcf332cd2d8">

Reply:
When did that happen? Works on my device for Ethereum Mainnet and also Goerli Testnet.

<img width="746" alt="Screenshot 2023-06-28 at 12 20 01 PM" src="https://github.com/trezor/trezor-suite/assets/66002635/361b5c74-8478-476c-8334-1fe63c829de4">

<img width="736" alt="Screenshot 2023-06-28 at 12 19 49 PM" src="https://github.com/trezor/trezor-suite/assets/66002635/83443890-24b6-4add-900a-d6e613da2b47">

Reply:
Can't reproduce your bug. When did that happen. Works on my device for Ethereum Mainnet and also Goerli Testnet.

<img width="748" alt="Screenshot 2023-06-28 at 12 31 41 PM" src="https://github.com/trezor/trezor-suite/assets/66002635/1ed7f4fe-de2b-4334-aa63-156cc2bd032c">

Reply:
Current gas price reported from Blockbook was `0.7983... gwei` as can be seen in console. 

```
const minFeeFromNetwork = new BigNumber(fromWei(info.levels[0].feePerUnit, 'gwei'),).integerValue(BigNumber.ROUND_FLOOR);
```

That means that because of `BigNumber.ROUND_FLOOR` the `minFeeFromNetwork` was `0`

Reply:
Hmm so maybe another check that if gas price is 0 from the network then also do plus(1)? Or what should be the gas price if it is 0?

Reply:
https://github.com/trezor/trezor-suite/blob/develop/suite-common/wallet-utils/src/sendFormUtils.ts#L179
```// blockbook/geth may return 0 in feePerUnit. if this happens set at least minFee```

Right now, when I check fee levels for goerli in Suite, it is `1544` wei so less than `1 gwei`

So `minFeeFromNetwork` is equal to `0`
`getMinFee` return `0` 

I am kinda confused from the logic, I cannot give you an advice without deep dive into the logic. However, `minFee` is modified to `0` and `feePerUnit` in fee `levels` too.
So the result is  ⬇️ 
![Screenshot 2023-06-29 at 11 21 37](https://github.com/trezor/trezor-suite/assets/33235762/87e45f51-f191-42da-b85a-982e13d63396)

Reply:
I see. I fixed the logic based on that. Can you try it? I think it should be fine now.